test/src/specs/sync/invalid_locator_size.rs

```diff
@@ -11,6 +11,9 @@ use log::info;
 
 pub struct InvalidLocatorSize;
 
+// test same value as ckb_sync::MAX_LOCATOR_SIZE;
+const MAX_LOCATOR_SIZE: usize = 101;
+
 impl Spec for InvalidLocatorSize {
```

Comment:
https://github.com/nervosnetwork/ckb/blob/63a699b65df1e542107956835c64eadc0d460a68/sync/src/lib.rs#L53

Reply:
This refractory does not affect  `MAX_LOCATOR_SIZE`, should remain unchanged

Reply:
This PR want to remove the dependence of `ckb-sync` crate in the integration test, 3 constants are copied to test code: RETRY_ASK_TX_TIMEOUT_INCREASE / BLOCK_DOWNLOAD_TIMEOUT / MAX_LOCATOR_SIZE, we may move them and more shared constants to a new crate (ckb-constants), and change `ckb-sync` and integration test to use this new crate in the future PR. 

I prefer to re-define it in test code regarding this PR, let's keep changes to a minimum.

Reply:
I insist on this should remain unchanged, since this PR does not move it.
I think this is the so-called minimum change.